docs/appendices/release-notes/unreleased.rst

```diff
@@ -197,6 +197,9 @@ Fixes
 .. stable branch. You can add a version label (`v/X.Y`) to the pull request for
 .. an automated mergify backport.
 
+- Fixed an issue which caused aggregations used in  ``INSERT INTO`` statements
+  returning null instead of the aggregation result.
```

Comment:
Is this a regression introduced with recent releases? If so, lets point that out please.